torch/distributed/fsdp/optim_utils.py

```diff
@@ -513,14 +511,15 @@ def _get_flat_param_to_fsdp_module(
     return flat_param_to_fsdp_module
 
 
-def _get_flat_param_id_to_param(
+def _get_param_id_to_param(
```

Comment:
Renaming this because it actually works for parameter ID to parameter in both the flattened and unflattened cases, as long as the keys and values are consistent.